app/views/businessactivities/involved_in_other_name.scala.html

```diff
@@ -37,63 +59,40 @@
 
     @form(f, controllers.businessactivities.routes.InvolvedInOtherController.post(edit)) {
 
-        @fieldset(
-            f = f("involvedInOther-fieldset"),
+
+        @fieldsetWithErrorGroup(
+            f = f("involvedInOther"),
             heading = "businessactivities.involved.other.title",
-            panel = false
+            section = "summary.businessactivities",
+            panel = false,
+            supportingContent = supportingContent
         ) {
-            @businessTypes.map { types =>
-                @if(types.size > 1) {
-                    <p>@Messages("businessactivities.confirm-activities.subtitle_4"):</p>
-
-                    <ul class="list list-bullet">
-                        @types.map { businessType =>
-                            <li>@businessType</li>
-                        }
-                    </ul>
-                } else {
-                    <p>@Messages("businessactivities.confirm-activities.subtitle_4") @types.head </p>
-                }
-            }
-
-            <p>@Messages("businessactivities.involved.other.hint")</p>
-            <ul class="list list-bullet">
-                <li>@Messages("businessactivities.involved.listline1")</li>
-                <li>@Messages("businessactivities.involved.listline2")</li>
-                <li>@Messages("businessactivities.involved.listline3")</li>
-            </ul>
-            @errorGroup(
-                field = f("involvedInOther")
+            @radio(
+                f = f("involvedInOther"),
+                labelText = "lbl.yes",
+                value = "true",
+                target = s"#$target"
+            )
+            @panel(
+              field = Some(f(s"$target")),
+              jsHidden = false,
+              id = target
             ) {
-                @radio(
-                    f = f("involvedInOther"),
-                    labelText = "lbl.yes",
-                    value = "true",
-                    target = s"#$target",
-                    inline = true
-                )
-                @panel(
-                  field = Some(f(s"$target")),
-                  jsHidden = false,
-                  id = target
-                ) {
-                    @textarea(
-                      field = f("details"),
-                      labelText = "businessactivities.confirm-activities.lbl.details",
-                      hintText = "businessactivities.involved.hint",
-                      bold = false,
-                      largeWidth = true,
-                      rows = "6",
-                      cols = "30"
-                    )
-                }
-                @radio(
-                    f = f("involvedInOther"),
-                    labelText = "lbl.no",
-                    value = "false",
-                    inline = true
+                @textarea(
```

Comment:
Out of curiosity. What does text area do? From the name it kind of insinuates it should be supportingContent but I could be wrong. I can't make out the html

Reply:
Would it not be more consistent to have all text/paragraph fields within the supportingContent area? Instead of having some views with textAreas inside the fieldset and some views with paragraph text inside supportingContent.

Reply:
Alright cheers